osrm/OSRM_nodejs_bindings.md

```diff
@@ -0,0 +1,265 @@
+# Making OSRM Matrix Computations with its NodeJS Bindings
+
+In this tutorial we would like to demonstrate the power of [Open Source Routing Machine](https://github.com/Project-OSRM/osrm-backend) utilizing its [NodeJS bindings](https://www.npmjs.com/package/osrm) without any overhead of running a service. 
+If it's the first time you have heard of bindings, just imagine them as an application programming interface (API) that provides glue code specifically made to allow a programming language to use a foreign library which is not native to that language.
+In this scenario our foreign library is OSRM written in C++ which is being consumed by JavaScript.
+Additionally, we would like to give you a small taste of how you could optimize for performance when requesting large square matrices.
+
+## Introduction
+
+[Open Source Routing Machine](https://github.com/Project-OSRM/osrm-backend) is a high performance open source routing software (BSD-2-Clause License) written in C++ and mainly designed to consume OpenStreetMap data. 
+
+For a comprehensive feature list and a more general overview of FOSS routing engines, read our [article](https://gis-ops.com/open-source-routing-engines-and-algorithms-an-overview/).
+
+While we are interested in the matrix (table) capabilities, you also have the option to use its other services, too.
+
+- [routing](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/nodejs/api.md#route)
+- [matrix](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/nodejs/api.md#table)
+- [nearest points](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/nodejs/api.md#nearest)
+- [map tiles](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/nodejs/api.md#tile)
+- [map matching](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/nodejs/api.md#match)
+- [traveling salesman problem](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/nodejs/api.md#trip)
+
+ > **Disclaimer**  
+ > Validity only confirmed for **Ubuntu 20.04** / **Mac OSX 10.15.6** and **[OSRM 5.26.0](https://www.npmjs.com/package/osrm)**.
+
+### Recommended
```

Comment:
IMO there's our standard `## Prerequisites` missing, that also talks about what to install before attempting the tutorial, e.g. npm/node (others?)